lib/deep-equal.test.js

```diff
@@ -365,39 +364,43 @@ describe("deepEqual", function() {
         assert.isFalse(checkDeep);
     });
 
-    it("returns false if object has different symbolic properties", function() {
-        var obj1 = {};
-        var obj2 = {};
-        obj1[symbol] = 42;
-        obj2[symbol] = 43;
-        var checkDeep = samsam.deepEqual(obj1, obj2);
-        assert.isFalse(checkDeep);
-    });
+    if (typeof Symbol === "symbol") {
```

Comment:
Can we skip these instead? This keeps the number of tests consistent and obvious. We should try to move away from this practice.

You're going to need to nest these tests in a `describe`. In the `before` of that describe if this check fails, call `this.skip();`.

Reply:
Ah, interesting. I didn't know this is possible. Will change as you suggested 👍

Reply:
Making this change even showed that I was _always_ skipping the tests because the conditional was wrong 🙄 Thank you for sharing this idea. Much better way of doing conditional tests 🙌